app/pages/index.tsx

```diff
@@ -22,119 +25,113 @@ function Index() {
   );
   console.log(JSON.parse(JSON.stringify(userData)));
 
+  const modalCheckboxRef = useRef<HTMLInputElement>(null);
+
+        // Need to refernce input newHouse modal to close the model on submit
+
   const handleCreateHousehold = async (event: any) => {
     event.preventDefault();
-    const { name } = event.target.elements;
+    const house_name = event.target.elements.house_name;
+    const house_desc = event.target.elements.house_desc;
+
+    var flag = false;
+    // Input Validation
+    if(house_name.value === "") {
+      house_name.classList.add("input-error")
+      console.log("INPUT_ERROR: House name is empty!");
+      flag =true;
+    }
+    if(house_desc.value.length>50) {
+      house_desc.classList.add("textarea-error")
+      console.log("INPUT_ERROR: Description must be less than 50 characters!");
+      flag=true;
```

Comment:
return;

Reply:
For this part, was thinking if user fails to both enter a name and enter a description greater than 50 words, the error prompt will show both instead of just the House name showing an error.